qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -17,11 +17,17 @@
 import importlib
 import os
 import sys
+from functools import partial
```

Comment:
Can we alphabetize this, please?

Reply:
PyCharm auto formats it like this, which I think is the PEP8 style format